.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Format codebase using ruff @johannaengland 09/09/2024
+07fcc0fa89e719a7cd00d36a3f12e3b552ac6ef2
```

Comment:
Code is added to this repo by squashing so this will not be correct.

Reply:
Yes, I am aware, I was not planning on merging it like this, I just added this commit to not forget about it later